gtsfm/frontend/verifier/degensac.py

```diff
@@ -23,23 +23,25 @@
 from gtsfm.common.keypoints import Keypoints
 from gtsfm.frontend.verifier.verifier_base import VerifierBase
 
-PIXEL_COORD_RANSAC_THRESH = 0.5
-
 logger = logger_utils.get_logger()
 
+MAX_TOLERATED_POLLUTION_INLIER_RATIO_EST_MODEL = 0.1
```

Comment:
Too long name, and its not clear what it means either. What is “tolerated polluted”? Is MIN_INLIER_CORRESPONDENCES_RATIO better? Also, would you like to move that to config?

Reply:
maybe `MIN_RATIO_OF_OUTPUT_TO_INPUT_POINTS`? if we want to be explicitly clear?

Reply:
it should probably go in the config, under verifier.

Reply:
i moved it to the config and renamed it to `min_allowed_inlier_ratio_est_model`